requirements.txt

```diff
@@ -1,15 +1,9 @@
-shortuuid==0.4.3
 pystache==0.5.4
-configobj==5.0.6
 wikipedia==1.4.0
 requests==2.13.0
-pyOpenSSL==16.2.0
```

Comment:
So I looked into that earlier, and requests depends on pyOpenSSL only if you use the `security` version of it, which would be requests@security. We don't use that, so we should be good there. What issues were you seeing where pyOpenSSL not being installed led to errors, and what platforms?